docs/index.md

```diff
@@ -0,0 +1,82 @@
+# Index
+
+Scistream enables high-speed memory-to-memory
+ data streaming in scientific environments.
+
+ We have a python implementation and a deployment solution compatible with docker and kubernetes.
+
+If you want to learn more about Scistream, please take a look at our [HPDC'22 paper](https://dl.acm.org/doi/abs/10.1145/3502181.3531475).
+
+## Documentation
+
+   - [Getting started](quickstart.md)
+   - [User Guide](user-guide/README.md)
+   - [Developer Guide](dev-guide/README.md)
+   - [Benchmarks](benchmarks/README.md)
+   - [About](about/README.md)
+
+## Project layout
+
+    poetry.lock         # Explicitly documents all Python dependencies using Poetry
+    pyproject.toml      # Python dependencie: Poetry equivalent to requirements.txt
+    mkdocs.yml          # Documentation configuration file.
+    docs/
+      index.md          # The documentation homepage.
+      ...               # Other markdown pages, images and other files.
+    deploy/
+      setup.sh          # Installation script
+    src/
+      proto/            # GRPC protocol specifications
+      appcontroller.py  # Application controller reference implementations
+      s2cs.py           # Scistream Control Server implementation
+      s2ds.py           # Scistream Dataplane server plugin
+      s2uc.py           # Scistream User Client implementation
+      utils.py          # Supporting code
+      ...               # other files
+    tests/
+      test_s2cs.py      # pytest tests
+      ...               # other tests
+
+### Service
+      The protocol should enable high-speed, memory-to-memory data streaming in scientific environments
+      by establishing streaming data channels between two remote facilities using our reference architecture:
+
+      ![alt text](figures/simple-arch.png "SciStream architecture")
```

Comment:
Call to the figures may not be rendering well. 
[scistream-simple-arch-with-software.pdf](https://github.com/scistream/scistream-proto/files/14317047/scistream-simple-arch-with-software.pdf)
Maybe we should use this newer image instead.